.github/workflows/ci.yml

```diff
@@ -1,30 +1,54 @@
-name: nf-core/cageseq CI
-# This workflow is triggered on pushes and PRs to the repository.
-# It runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
-on: [push, pull_request]
+name: nf-core CI
+# This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+  release:
+    types: [published]
 
 jobs:
   test:
+    name: Run workflow tests
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/cageseq') }}
+    runs-on: ubuntu-latest
     env:
       NXF_VER: ${{ matrix.nxf_ver }}
       NXF_ANSI_LOG: false
-    runs-on: ubuntu-latest
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
         nxf_ver: ['19.10.0', '']
-        aligner: ["--aligner 'bowtie'", "--aligner 'star'"]
-        options: ['--trimming false','--cutEcoP false','--cutLinker false','--cutG false','--cutArtifacts false', '--star_index false','--bowtie_index false']
+        aligner: ["--aligner 'bowtie1'", "--aligner 'star'"]
```

Comment:
Do you need the single quotes around `bowtie1` and `star`?

Reply:
hmm, yes, they might not be needed.